spec/requests/alert_definitions_spec.rb

```diff
@@ -306,4 +306,77 @@
     expect(alert_definition_profiles.first.reload.description).to eq("Updated Test Alert Profile 1")
     expect(alert_definition_profiles.second.reload.description).to eq("Updated Test Alert Profile 2")
   end
+
+  it "gets assignments for an alert definition profile" do
+    api_basic_authorize action_identifier(:alert_definition_profiles, :read, :resource_actions, :get)
+    alert_definition_profile = FactoryGirl.create(:miq_alert_set)
+    enterprise = FactoryGirl.create(:miq_enterprise)
+    enterprise_href = api_enterprise_url(nil, enterprise)
+    alert_definition_profile.assign_to_objects(enterprise)
+
+    get(api_alert_definition_profile_url(nil, alert_definition_profile))
+
+    expect(response).to have_http_status(:ok)
+    expect(response.parsed_body["get_assigned_tos"]["objects"]).to include(a_hash_including("href"=>enterprise_href))
```

Comment:
it would be nicer to have the api return 
s/get_assigned_tos/assigned_to/
(unless there is a convention for assignments i'm not aware of)

Reply:
I agree it would be much nicer, but from what I understood when talking with @jntullo about this on gitter when I wrote this PR, it's not possible to for the API to rename anything defined as `additional_attributes` - the name has to be identical to the name of the method in the model.